CMakeLists.txt

```diff
@@ -191,9 +199,19 @@ if(${BUILD_ATTACH_IMPL_EXAMPLE})
 endif()
 
 # benchmark that requires bpftime libraries
+if(${BPFTIME_BUILD_WITH_LIBBPF})
+# Currently benchmark is using libbpf
 add_subdirectory(benchmark)
+endif()
 
 
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
 
-install(TARGETS bpftime-agent bpftime_text_segment_transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ~/.bpftime)
+set(DEST_DIR "$ENV{HOME}/.bpftime")
+if(${BPFTIME_BUILD_WITH_LIBBPF})
+    install(TARGETS bpftime-agent bpftime_text_segment_transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ${DEST_DIR})
```

Comment:
Syscall server shouldn't be excluded if libbpf was not enabled. bpftime relies on syscall server to work, by injecting syscall server to a userspace eBPF program. If some parts of syscall server won't work on macos, try adapting or disabling these parts

Reply:
okay, will have to add those changes then.